msteams-platform/resources/schema/localization-schema.md

```diff
@@ -11,8 +11,89 @@ ms.date: 04/14/2025
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.parameters\\[[0-4]\\]\\.title`|String|32|Specifies a localized value for the [composeExtensions.commands.parameters.title property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].parameters[0-4].title`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.parameters\\[[0-4]\\]\\.description`|String|128|Specifies a localized value for the [composeExtensions.commands.parameters.description property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].parameters[0-4].description`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.parameters\\[[0-4]\\]\\.value`|String|512|Specifies a localized value for the [composeExtensions.commands.parameters.value property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].parameters[0-4].value`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.parameters\\[[0-4]\\]\\.choices\\[[0-9]\\]\\.title`|String|128|Specifies a localized value for the [composeExtensions.commands.parameters.choices.title property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].parameters[0-4].choices[0-9].title`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.samplePrompts\\[[0-4]\\]\\.text`|String|128|Specifies a localized value for the [composeExtensions.commands.samplePrompts.text property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].samplePrompts[0-4].text`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.taskInfo\\.title`|String|64|Specifies a localized value for the [composeExtensions.commands.taskInfo.title property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].taskInfo.title`.||
+|`activities.activityTypes.description`|String|128|Specifies a localized value for the [activities.activityTypes.description property](manifest-schema.md#activitiesactivitytypes). The property name should be a JSON path expression in the following form: activities.`activityTypes[0-127].description`.||
+|`activities.activityTypes.templateText`|String|128|Specifies a localized value for the [activities.activityTypes.templateText property](manifest-schema.md#activitiesactivitytypes). The property name should be a JSON path expression in the following form: `activities.activityTypes[0-127].templateText`.||
```

Comment:
activities.activityTypes.templateText 

Here activityTypes is actually an array so is it correct or we need to define like: activities.activityTypes\\[[0-9]\\]\\.templateText

I understand that we have already added the description like this: `activities.activityTypes[0-127].templateText`

Reply:
The correct regex value for ``activities.activityTypes.templateText` is `activities.activityTypes\\[\\b([0-9]|[1-8][0-9]|9[0-9]|1[01][0-9]|12[0-7])\\b]\\.templateText`, this does not render well in the doc, please see the preview attached.

![image](https://github.com/user-attachments/assets/d605bae9-4ec5-4cba-8b3a-f474d9b466d5)